src/Hooks/FieldStorageConfigEditFormAlter.php

```diff
@@ -0,0 +1,81 @@
+<?php
+
+namespace Drupal\file_encrypt\Hooks;
+
+use Drupal\Core\Field\FieldTypePluginManagerInterface;
+use Drupal\Core\Form\FormStateInterface;
+use Drupal\Core\StringTranslation\StringTranslationTrait;
+use Drupal\encrypt\EncryptionProfileManagerInterface;
+use Drupal\field\FieldStorageConfigInterface;
+use Drupal\file\Plugin\Field\FieldType\FileItem;
+
+class FieldStorageConfigEditFormAlter {
+
+  use StringTranslationTrait;
```

Comment:
Oh, this is a [trait](https://secure.php.net/manual/de/language.oop5.traits.php) , its just how it is